sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -85,25 +85,22 @@ export type HrTime = [number, number];
 // @public
 export interface Link {
     attributes?: SpanAttributes;
-    context: LinkContext;
+    context: SpanContext;
 }
 
 // @public
-export type LinkContext = {
-    traceId: string;
-    spanId: string;
-};
+export type LinkContext = SpanContext;
```

Comment:
This might not be needed at all, I'll take a look. [1.0.0-rc3](https://unpkg.com/browse/@opentelemetry/api@1.0.0-rc.3/build/src/trace/link.d.ts) you can see that `LinkContext` was a subset of `SpanContext`... in [0.20.0](https://unpkg.com/browse/@opentelemetry/api@0.20.0/build/src/trace/link.d.ts) it was changed to just be the `SpanContext`